core/src/main/java/org/geysermc/connector/network/session/GeyserSession.java

```diff
@@ -47,10 +47,6 @@ public AuthData getAuthData() {
         return new AuthData(this.handle.getAuthData());
     }
 
-    public boolean isMicrosoftAccount() {
-        return this.handle.isMicrosoftAccount();
```

Comment:
it's not like this is supported but couldn't we just always return true

Reply:
It's the connector that imo should long be removed.. I wouldn't work around it tbh

Reply:
Then can we propose removing it now

Reply:
yeeted connector entirely in https://github.com/GeyserMC/Geyser/pull/4147/commits/4e07d7eb208108902101001d5588b34354629676